libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt

```diff
@@ -146,6 +164,46 @@ class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJa
         }
     }
 
+    @ReactMethod
+    fun signMessage(reqData: ReadableMap, message: String, promise: Promise) {
+        executor.execute {
+            val signMessageRequest = asSignMessageRequest(reqData)
+            if signMessageRequest == null {
+                 promise.reject(TAG, "Invalid reqData")
+            } else {
+                try {
+                    val response = getBreezServices().sign_message(signMessageRequest)
+                    val signedMessage = response.signed_message
+                    promise.resolve(signedMessage)
```

Comment:
The SignMessageResponse object should be returned here, you can add a mapping function to `BreezSDKMapper.kt`

```
fun readableMapOf(signMessageResponse: SignMessageResponse): ReadableMap {
    return readableMapOf(
            "signedMessage" to signMessageResponse.signedMessage
    )
}
```